common/lib/xmodule/xmodule/course_module.py

```diff
@@ -184,6 +184,11 @@ class CourseFields(object):
         help=_("Enter the date you want to advertise as the course start date, if this date is different from the set start date. To advertise the set start date, enter null."),
         scope=Scope.settings
     )
+    pre_requisite_course = String(
```

Comment:
Sorry to keep saying the same thing over and over... but.... changing this from a String to a List later will be difficult. Why not make this a List now? FYI, @mattdrayer